pkg/kubeauth/aws/aws.go

```
@@ -229,25 +228,29 @@ func GetRestConfigForAllClusters(awsConfig *api.AWSConfig) (map[string]*kubeauth
 		})
 		if err != nil {
 			logrus.Errorf("Failed to describe cluster %v: %v", *clusterName, err)
-			return nil, err
+			continue
```

Is it not better to return error? If the he does not have describeCluster permission, we might be return empty list. That should not indicate to the user that he does not have cluster.

---

Usually, if he is able to describe one cluster he should be able to others also from a permission point of view (as permission is at the IAM user level). Here the chances of error could be more on some issues with API for a particular cluster.  The thinking was if one Describe cluster fails, didn't want to throw an error or empty list as the user would be confused. 
We can get feedback from the test and Niranjan and see if this is ok or fail the operation itself.